front/components/actions/mcp/RemoteMCPForm.tsx

```diff
@@ -31,6 +36,7 @@ interface RemoteMCPFormProps {
 const MCPFormSchema = z.object({
   name: z.string().min(1, "Name is required."),
   description: z.string().min(1, "Description is required."),
+  icon: z.enum(["rocket", "command"], { required_error: "Icon is required." }),
```

Comment:
i think you can take the list of values from MCP_SERVER_ICONS

Reply:
yeah true, will fix :)